### PR TITLE
Yin2021 Center Based

### PR DESCRIPTION
**Pull request by jchavezberkeley**
I'm very sorry this was late. I will update this as soon as I can on Saturday to get it reviewed by Sunday

### CODE REVIEW

File: _summaries/yin2020center-based_2.md [hunk 1]
```diff
@@ -1,19 +1,48 @@
...
+
+<img src="yin2020center-based_b.png">
+
+On the nuScenes dataset, CenterPoint outperforms CBGS with multi-scale inputs and multi-model ensemble by 5.2% mAP and 2.2% NDS.
+
+<img src="yin2020center-based_c.png">
+
 * What interesting variants are explored?
 
+One-stage and two-stage CenterPoint were compared as part of the ablation study. Other things that were looked at were Voxel Set abstraction and radial basis function interpolation.
```
Comment (SShivang) [inline _summaries/yin2020center-based_2.md:43]: Here, you could include the table with the ablation results.

File: _summaries/yin2020center-based_2.md [hunk 2]
```diff
@@ -1,19 +1,48 @@
...
+The regression heads help provide "full state" information of 3D bounding boxes. Regression to log size also helps better handle boxes of different shapes. Then, at inference time, properties are extracted by indexing into the regression head outputs at each object's peak location.
+
+Two-dimensional velocity estimation is learned as another regression output. This is uses to predict the difference in object position between the current and past frame. 
+
+Finally, CenterPoint combines all the heatmap and regression losses and optimizes then before going onto the second phase.
+
+The second phase of CenterPoint extracts additional point-features from the backbone. Extracted point-features are passed through an MLP, and then, confidence scores and box refinements are predicted.
+
 * How well does the paper perform?
+
```
Comment (SShivang) [inline _summaries/yin2020center-based_2.md:32]: How does this work improve on the previous work? What is unique about this methodology?

File: _summaries/yin2020center-based_2.md [hunk 3]
```diff
@@ -1,19 +1,48 @@
...
+<img src="yin2020center-based_a.png" width="90%">
+
+During the first phase, the CenterPoint paper mentions 3 types of heads: center headmap heads, regression heads, velocity heads. The center-head's job is to produce a heatmap at the center location of any detected object. 
+
+The regression heads help provide "full state" information of 3D bounding boxes. Regression to log size also helps better handle boxes of different shapes. Then, at inference time, properties are extracted by indexing into the regression head outputs at each object's peak location.
+
+Two-dimensional velocity estimation is learned as another regression output. This is uses to predict the difference in object position between the current and past frame. 
+
+Finally, CenterPoint combines all the heatmap and regression losses and optimizes then before going onto the second phase.
```
Comment (SShivang) [inline _summaries/yin2020center-based_2.md:27]: Nice description of the method.

File: _summaries/yin2020center-based_2.md (hunk 1 above)
Comment (jchavezberkeley) [reply _summaries/yin2020center-based_2.md:43]: Yes I can

**Review by zhaoyue-zephyrus — APPROVED** [review]

**Pull request merged**